src/dashboard/src/media/js/ingest.js

```diff
@@ -331,13 +331,6 @@ $(function()
             window.location.href = '/ingest/' + this.model.sip.get('uuid') + '/upload/atk/';
           }
 
-          // redirect to object/resource mapping pages
```

Comment:
It'd probably be good to ask Justin/Sarah about this, but given the circumstances this is probably the best compromise. Maybe we could make the way you open the matching GUI manually more obvious to compensate?